app/src/pages/AbstractStreamEditPage/AccessControlSection/index.tsx

```diff
@@ -15,14 +15,14 @@ const AccessControlSection: React.FunctionComponent<Props> = ({ disabled: disabl
     const disabled = disabledProp || !canEdit
 
     return (
-        <TOCSection id="accessControl" title="Access control">
+        <Section title="Access control">
```

Comment:
There's more of this here. `TOCSection` (gone) got simplified and is now `Section`. Less moving parts.

Similar simplification happened in all other stream page `*Section` components.